packages/@lwc/engine-core/src/patches/detect-non-standard-aria.ts

```diff
@@ -62,7 +62,7 @@ function checkAndReportViolation(elm: Element, prop: string, isSetter: boolean,
         if (process.env.NODE_ENV !== 'production') {
             logWarnOnce(
                 `Element <${elm.tagName.toLowerCase()}> ` +
-                    (isUndefined(vm) ? '' : `owned by <${vm.elm.tagName.toLowerCase()}> `) +
+                    (isUndefined(vm) ? '' : `owned by <${vm.tagName.toLowerCase()}> `) +
```

Comment:
Updated this to be consistent with everywhere else since this is the only place we do `vm.elm.tagName`.